setup.py

```diff
@@ -22,7 +22,15 @@
 
 name = "google-cloud-bigquery"
 description = "Google BigQuery API client library"
-version = "2.1.0"
+
+
+package_root = os.path.abspath(os.path.dirname(__file__))
```

Comment:
if you migrate this addition down to ~ln 88, package_root is already defined and can be reused